app/helpers/signs_helper.rb

```diff
@@ -17,4 +17,12 @@ def sign_status_orb(sign)
     orb_class = sign.private? ? 'sign-private' : sign.active? ? 'sign-active' : 'sign-inactive'
```

Comment:
It's being called from a slides view. This is a case where you want to use [Schelling point logic](https://en.wikipedia.org/wiki/Focal_point_(game_theory)). If you were a fresh developer coming across this helper in context, where would be the most logical place to look for it? I would probably head for `slides_helper.rb`. I think context and convention dictate that expecation.